Samples/Compute/ConvertVirtualMachineToManagedDisks.cs

```diff
@@ -52,7 +52,7 @@ public static void RunSample(IAzure azure)
                             .WithNewVhd(50)
                             .WithLun(2)
                             .Attach()
-                        .WithSize(VirtualMachineSizeTypes.StandardD3V2)
+                        .WithSize(VirtualMachineSizeTypes.Parse("Standard_D2a_v4"))
```

Comment:
What's the cause?

Reply:
For test purpose, we want to move vm size into AMD based.